### PR TITLE
added section on interop

### PR DESCRIPTION
**Pull request by jandrieu**
per https://github.com/w3c-ccg/did-wg-charter/pull/58

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/pull/234.html" title="Last updated on Aug 8, 2019, 8:20 PM UTC (e0e68d0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/did-spec/234/31ae2ee...e0e68d0.html" title="Last updated on Aug 8, 2019, 8:20 PM UTC (e0e68d0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -468,8 +468,25 @@ <h2>
         </tbody>
       </table>
     </section>
-  </section>
 
+     <section>
+      <h3>Interoperability</h3>
```
Comment (peacekeeper) [inline index.html]: Should this be called "Conformance" instead of "Interoperability"?

File: index.html [hunk 2]
```diff
@@ -468,8 +468,25 @@ <h2>
...
+          <p>Interoperability of implementations for DIDs and DID Documents will be tested by evaluating an implementation's ability to create and parse DIDs and DID Documents that conform to the specification. Interoperability for DID methods will be determined by evaluating each DID method's specification to determine, at a minimum, </p>
+          <ol>
+              <li> the DID method name is unique and not used by an existing, incompatible DID method, </li>
+              <li> the required operations are supported,</li>
+              <li> operations requiring descriptions are described,</li>
+              <li> the specification is specific, detailed, and complete enough for independent implementation, and</li>
+              <li> the specification contains sections describing security and privacy considerations.</li>
+          </ol>
+              
+          <p>The working group's test suite will test all normative requirements for DIDs and DID Documents. Human evaluators will need to read DID Method specifications to evaluate conformance.</p>
```
Comment (peacekeeper) [inline index.html]: Feels a bit strange to have text in the spec that talks about something the WG will do, no?

File: index.html (hunk 1 above)
Comment (jandrieu) [reply index.html]: Maybe. Although then I think there should be more text in there. The question of interoperability is about constraint on what "conformance" means and I expect there should be more explaining the full scope of conformance if that's the section heading.

File: index.html [hunk 3]
```diff
@@ -468,8 +468,25 @@ <h2>
         </tbody>
       </table>
     </section>
-  </section>
 
+     <section>
```
Comment (davidlehn) [inline index.html]:
```suggestion
    <section>
```

File: index.html [hunk 4]
```diff
@@ -468,8 +468,25 @@ <h2>
...
+      <h3>Interoperability</h3>
+          <p>Interoperability of implementations for DIDs and DID Documents will be tested by evaluating an implementation's ability to create and parse DIDs and DID Documents that conform to the specification. Interoperability for DID methods will be determined by evaluating each DID method's specification to determine, at a minimum, </p>
```
Comment (davidlehn) [inline index.html]: The markup indents should be using 2 spaces per level.  Also would be nice to wrap paragraphs at ~80 chars like the current format.

File: index.html [hunk 5]
```diff
@@ -468,8 +468,25 @@ <h2>
...
+          <ol>
+              <li> the DID method name is unique and not used by an existing, incompatible DID method, </li>
+              <li> the required operations are supported,</li>
+              <li> operations requiring descriptions are described,</li>
+              <li> the specification is specific, detailed, and complete enough for independent implementation, and</li>
+              <li> the specification contains sections describing security and privacy considerations.</li>
+          </ol>
+              
+          <p>The working group's test suite will test all normative requirements for DIDs and DID Documents. Human evaluators will need to read DID Method specifications to evaluate conformance.</p>
```
Comment (dlongley) [inline index.html]:
```suggestion
          <p>The working group's test suite will test all normative requirements for DIDs and DID Documents. Human evaluators will need to read DID Method specifications to evaluate conformance.</p>
```

**Pull request merged**